app/src/main/java/org/hackillinois/android/view/ScannerFragment.kt

```diff
@@ -61,25 +61,22 @@ class ScannerFragment : Fragment() {
                     val userId: String = getUserIdFromQrString(it.text)
                     viewModel.checkUserIntoEvent(event, userId, view.staffOverrideSwitch.isChecked)
                     Handler(Looper.getMainLooper()).post {
-                        Toast.makeText(context, "Sending request to API. Please wait for a response", Toast.LENGTH_LONG).show()
+                        Toast.makeText(context, "Please wait", Toast.LENGTH_LONG).show()
                     }
                 }
                 errorCallback = ErrorCallback {
                     Handler(Looper.getMainLooper()).post {
-                        Toast.makeText(context, "Camera initialization error: ${it.message}", Toast.LENGTH_LONG).show()
+                        Toast.makeText(context, "${it.message}", Toast.LENGTH_LONG).show()
                     }
                 }
             }
 
             view.codeScannerView.setOnClickListener { codeScanner.startPreview() }
 
             // get camera permission
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-                if (context.checkSelfPermission(Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M &&
```

Comment:
You could put both sides of the && into boolean variables to prevent really long lines